overlord/state/notices.go

```diff
@@ -192,12 +192,13 @@ const (
 	WarningNotice NoticeType = "warning"
 )
 
+var ValidNoticeTypes = map[NoticeType]bool{
```

Comment:
this probably could be just called NoticeTypes  and probably merits a short doc comment.

Reply:
Updated, Thank you!